addon/templates/components/frost-fixed-table.hbs

```diff
@@ -84,6 +84,7 @@
     }}
       {{#each items as |item index|}}
         {{frost-table-row
+          class=(if isSelectable 'selectable' '')
```

Comment:
....but we want the middle and right section rows to have this class so that the cursor pointer changes